.github/workflows/preview.yml

```diff
@@ -3,43 +3,91 @@ name: Publish Preview release
 on:
   schedule:
     # run every day at 00:15 UTC to avoid high load times at the start of every hour
-    - cron: '15 0 * * *'
+    - cron: "15 0 * * *"
```

Comment:
Do we want to use single or double quotes in yaml files? Prettier is formatting the file with single quotes for me.

Reply:
I use the YAML extension, not prettier, to format yaml and that uses double quote by default. If we already have an existing formatting convention for single quotes I can change the settings to use single instead.